Index: rocksdb-6.20.3/CMakeLists.txt
===================================================================
--- rocksdb-6.20.3.orig/CMakeLists.txt	2021-07-03 19:24:25.461922836 +0800
+++ rocksdb-6.20.3/CMakeLists.txt	2021-07-03 19:24:34.665934673 +0800
@@ -1344,7 +1344,7 @@
   add_executable(table_reader_bench${ARTIFACT_SUFFIX}
     table/table_reader_bench.cc)
   target_link_libraries(table_reader_bench${ARTIFACT_SUFFIX}
-    ${ROCKSDB_LIB} testharness ${GFLAGS_LIB})
+    ${ROCKSDB_LIB} ${GFLAGS_LIB})
 
   add_executable(filter_bench${ARTIFACT_SUFFIX}
     util/filter_bench.cc)
