Index: rocksdb-6.10.2/third-party/gtest-1.8.1/fused-src/gtest/CMakeLists.txt
===================================================================
--- rocksdb-6.10.2.orig/third-party/gtest-1.8.1/fused-src/gtest/CMakeLists.txt
+++ rocksdb-6.10.2/third-party/gtest-1.8.1/fused-src/gtest/CMakeLists.txt
@@ -1 +1,3 @@
+find_package(Threads REQUIRED)
 add_library(gtest gtest-all.cc)
+target_link_libraries(gtest Threads::Threads)

